api/provider/aws/environment_delete.go

```diff
@@ -53,6 +59,42 @@ func (e *EnvironmentProvider) Delete(environmentID string) error {
 	return nil
 }
 
+func (e *EnvironmentProvider) checkEnvironmentDependencies(environmentID string) error {
+	deleteEntity := func(entity string) error {
+		tags, err := e.TagStore.SelectByType(entity)
+		if err != nil {
+			return err
+		}
+
+		dependentTags := tags.WithKey("environment_id").WithValue(environmentID)
+		if len(dependentTags) > 0 {
+			IDs := []string{}
```

Comment:
even with acronyms, it's best practice to always have lowercase variable names - maybe `entityIDs`?